### PR TITLE
7.31c and nothing else

### PR DESCRIPTION
**Pull request by TheUnholyWrath**
[7.31c](https://www.dota2.com/patches/7.31c)

### CODE REVIEW

File: game/scripts/npc/abilities/bane_brain_sap.txt [hunk 1]
```diff
@@ -37,8 +37,8 @@
     {
       "brain_sap_damage" //OAA
       {
-        "value"                                           "75 150 225 300 600 900"
-        "special_bonus_unique_bane_2"                     "+300"
+        "value"                                           "100 200 300 400 800 1200"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/bane_brain_sap.txt:40]: why?

File: game/scripts/npc/abilities/lone_druid_spirit_bear.txt [hunk 2]
```diff
@@ -57,7 +57,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "backlash_damage"                                 "20 30 40 50 60 70" //OAA
+        "backlash_damage"                                 "20.0"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/lone_druid_spirit_bear.txt:60]: why?

File: game/scripts/npc/abilities/talents/storm_spirit_talent6.txt [hunk 3]
```diff
@@ -18,7 +18,7 @@
       "01"
       {
         "var_type"        "FIELD_INTEGER"
-        "value"           "250" //OAA
+        "value"           "125" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/talents/storm_spirit_talent6.txt:21]: Sadge

File: game/scripts/npc/abilities/undying_decay.txt [hunk 4]
```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "10 8 6 4 4 4"
+    "AbilityCooldown"                                     "8 7 6 5 4.5 4" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/undying_decay.txt:29]: I was thinking that str steal needs a rescale, not this.

File: game/scripts/npc/items/custom/item_devastator_1.txt [hunk 5]
```diff
@@ -53,7 +53,7 @@
     "AbilityCastPoint"                                    "0"
 
     "AbilitySharedCooldown"                               "devastator"
-    "AbilityCooldown"                                     "7"
+    "AbilityCooldown"                                     "9 8 7 6 5"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/items/custom/item_devastator_1.txt:56]: I would put 7s cd as min cd. 5s sounds bonkers.

File: game/scripts/npc/abilities/invoker_chaos_meteor.txt [hunk 6]
```diff
@@ -19,7 +19,7 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastRange"                                    "700"
     "AbilityCastPoint"                                    "0.05"
-    "AbilityCooldown"                                     "35" //OAA
+    "AbilityCooldown"                                     "40" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/invoker_chaos_meteor.txt:22]: I would nerf the talent that increases meteor damage by a percentage, but maybe this is alright too.

**Review by DarkoniusXNG — CHANGES_REQUESTED** [review]
If you don't want to change the things I marked, at least explain your thoughts please.

File: game/scripts/npc/abilities/bane_brain_sap.txt (hunk 1 above)
Comment (TheUnholyWrath) [reply game/scripts/npc/abilities/bane_brain_sap.txt:40]: +300 Brain Sap damage looked like a crap talent, I wanted to make it +400. 400 = 400. Bane bad damage good

File: game/scripts/npc/abilities/talents/storm_spirit_talent6.txt (hunk 3 above)
Comment (TheUnholyWrath) [reply game/scripts/npc/abilities/talents/storm_spirit_talent6.txt:21]: 200% of the level 4 damage as a talent is kinda yikes

File: game/scripts/npc/abilities/lone_druid_spirit_bear.txt (hunk 2 above)
Comment (TheUnholyWrath) [reply game/scripts/npc/abilities/lone_druid_spirit_bear.txt:60]: It said field float so I added a .0

File: game/scripts/npc/abilities/undying_decay.txt (hunk 4 above)
Comment (TheUnholyWrath) [reply game/scripts/npc/abilities/undying_decay.txt:29]: Yeah something is wrong with decay, I wasn't sure what it was. Str steal is integer values so it seemed weird to change it

**Comment by lovesmuffins** [comment]
rubick and morph changes r bad

File: game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt [hunk 7]
```diff
@@ -41,7 +41,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage_base"                                     "55 60 65 70 140 280" //OAA
+        "damage_base"                                     "40 60 80 120 240 480" //OAA
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt:44]: Adaptive strike agi early on is already hot garbage and morph is not at all a good early game hero that would deserve this nerf.

File: game/scripts/npc/abilities/rubick_spell_steal.txt [hunk 8]
```diff
@@ -46,7 +46,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "180.0 240.0 300.0 360.0 420.0"
+        "duration"                                        "9999.0" //OAA
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/rubick_spell_steal.txt:49]: no. just no.

File: game/scripts/npc/abilities/lone_druid_spirit_bear.txt (hunk 2 above)
Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/lone_druid_spirit_bear.txt:60]: I meant why buff Lone Druid?

File: game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt (hunk 7 above)
Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt:44]: It's a lvl 1 nerf. Dont be dramatic.

File: game/scripts/npc/abilities/lone_druid_spirit_bear.txt (hunk 2 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/lone_druid_spirit_bear.txt:60]: good change, backlash dmg is dumb anyways @DarkoniusXNG

**Pull request merged**